test-conftest

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -o errexit
+trap 'echo "Aborting due to errexit on line $LINENO. Exit code: $?" >&2' ERR
+set -o errtrace
+set -o pipefail
+
+conftest test --trace --policy policy/argocd_app.rego . 
```

Comment:
You don't have to specify the `--policy`. By default all policies in the `policy` folder are executed.

Also the `.` at the end, doesn't work for me, I must specify the `**/*.yaml` instead.

Reply:
i like it explicit, thats why --trace